### PR TITLE
Add --type-only option.

### PR DESCRIPTION
**Pull request by lacolaco**
If Google API uses names reserved in typescript, compilation errors occur.
e.g. `get`, `delete`
This option allows me use only interface declarations.

### CODE REVIEW

File: lib/plugin/lib/walker.ts [hunk 1]
```diff
@@ -25,16 +25,19 @@ function walk(plugin:IPlugin, process:Process, root:Root) {
...
+		walkDefinition(plugin, process, root, typeOnly);
 	}
 }
 
-function walkDefinition(plugin:IPlugin, process:Process, root:Root) {
-	root.resources.forEach(resource => walkResource(plugin, process, resource));
-	root.methods.forEach(method => walkMethod(plugin, process, method));
+function walkDefinition(plugin:IPlugin, process:Process, root:Root, typeOnly = false) {
+	if(!typeOnly) {
+		console.log("HIT");
```
Comment (vvakame) [inline lib/plugin/lib/walker.ts]: What's this?

**Comment by vvakame** [comment]
please separate to 3 PR.
1. add `--type-only`, 2. switch to dtsm, 3. update dependencies.

**Comment by laco0416** [comment]
Sorry, i fixed it.
please review again!

**Comment by vvakame** [comment]
@laco0416 thanks mate!

**Pull request merged**